deploy-manage/deploy/cloud-enterprise.md

```diff
@@ -1,27 +1,56 @@
 ---
-mapped_urls:
-  - https://www.elastic.co/guide/en/cloud-enterprise/current/Elastic-Cloud-Enterprise-overview.html
-  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-administering-ece.html
+applies:
+  ece: all
 ---
 
-# Elastic Cloud Enterprise
+# Elastic Cloud Enterprise [Elastic-Cloud-Enterprise-overview]
 
-% What needs to be done: Refine
+{{ece}} (ECE) is an Elastic self-managed solution for deploying, orchestrating, and managing {{es}} clusters at scale. It provides a centralized platform that allows organizations to run {{es}}, {{kib}}, and other {{stack}} components across multiple machines.
 
-% GitHub issue: https://github.com/elastic/docs-projects/issues/339
+Refer to [](./cloud-enterprise/ece-overview.md) for a detailed introduction to ECE, including its features, use cases, and architecture.
 
-% Scope notes: Ensure the landing page makes sense and its aligned with the section overview and the overview about orchestators. What content should be in deployment types overview or in the main overview and what in the ECE landing page...
+::::{tip}
+If you are looking for a solution to orchestrate and manage {{es}} clusters natively on Kubernetes, consider using [Elastic Cloud on Kubernetes (ECK)](./cloud-on-k8s.md) instead of ECE. ECK enables you to orchestrate Elastic Stack applications seamlessly on Kubernetes, leveraging it as the underlying platform for deployment, scaling, and lifecycle management.
+::::
 
-% Use migrated content from existing pages that map to this page:
+% should we use a L2 heading here or just continue?
+## Section overview
 
-% - [ ] ./raw-migrated-files/cloud/cloud-enterprise/Elastic-Cloud-Enterprise-overview.md
-%      Notes: 2 child docs
-% - [ ] ./raw-migrated-files/cloud/cloud-enterprise/ece-administering-ece.md
-%      Notes: redirect only
+This section focuses on deploying ECE and the orchestrating and configuring {{es}} clusters, also referred to as `deployments`.
 
-⚠️ **This page is a work in progress.** ⚠️
+In ECE, a deployment is a managed {{stack}} environment that provides users with an {{es}} cluster along with supporting components such as {{kib}} and other optional services like APM and Fleet.
 
-The documentation team is working to combine content pulled from the following pages:
+This section covers the following tasks:
 
-* [/raw-migrated-files/cloud/cloud-enterprise/Elastic-Cloud-Enterprise-overview.md](/raw-migrated-files/cloud/cloud-enterprise/Elastic-Cloud-Enterprise-overview.md)
-* [/raw-migrated-files/cloud/cloud-enterprise/ece-administering-ece.md](/raw-migrated-files/cloud/cloud-enterprise/ece-administering-ece.md)
\ No newline at end of file
+* [Deploy ECE](./cloud-enterprise/deploy-an-orchestrator.md)
```

Comment:
might want to specify that this is the orchestrator / installation - this always confused me in the original doc

Reply:
Done it